distributed/comm/tests/test_comms.py

```diff
@@ -769,6 +769,10 @@ async def test_tcp_comm_closed_explicit(tcp):
     await check_comm_closed_explicit("tcp://127.0.0.1")
 
 
+@pytest.mark.xfail(
+    sys.version_info[:2] == (3, 7),
```

Comment:
Should we assign this to a variable like `py37`? Wondering if we will spot other things like this issue where we similarly want to skip

Reply:
We can do that then if the need arises. No need now.